src/Column/index.jsx

```diff
@@ -96,6 +96,7 @@ Column.propTypes =
         'top',
         'middle',
         'bottom',
+        'space-around'
```

Comment:
This should not be here. Beware of `gaa` :smile:

Reply:
Yeah sorry about that :D